### DISCUSSION TITLE
SpiralLinux Cinnamon edition is fantastic!

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#354] [General]
Can't remember the last time i woke up in the a.m. wanting to get into an operating system!
Have tried a bunch over the years, but this SpiralLinux Cinnamon is quite pleasant!
For some reason, find it better than LinuxMint's why?
Stable, lightweight (at par with XFCE), snappy and just all around pleasing to work with.
Deserves to be in the top 5 at Distrowatch! :1st_place_medal:

**Comment by dacomboman** [comment]
Still dabbing around with SpiralLinux Cinnamon (almost a record!)
Just want to add how good sound comes out from hardwired cheap speakers! :+1:

**Reply by geckolinux** [reply]
Really glad to hear it's working well for your needs and preferences, thanks for saying so!
